inception-project/inception
# #277 Import/export external search settings

**What's in the PR**
* import and export functionality for a project covers external document repositories
* resolves #277 

**How to test manually**
* create a project with external document repositories
* export project and re-import into new project
* check whether external document repositories in new project match the original project

**Automatic testing**

**Documentation**
